src/examples/notifications/notifications/NotificationDefault.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Notification, Title, Close } from '@zendeskgarden/react-notifications';
+
+const Example = () => (
+  <Notification type="warning">
+    <Title>Warning</Title>
```

Comment:
Review warning example. No closing punctuation.

<Title>Soggy cactus</Title>
Let the soil dry out before watering again